src/app/modules/rup/components/elementos/FormulaBase.html

```diff
@@ -17,48 +17,53 @@
-                                [registro]="registro.registros[i]" [soloValores]="soloValores" (change)="emitChange2()"
-                                (ejecutarAccion)="emitEjecutarAccion($event)" [habilitado]="valorManual">
+                             [paciente]="paciente" [params]="item.params" [prestacion]="prestacion"
+                             [registro]="registro.registros[i]" [soloValores]="soloValores" (change)="emitChange2()"
+                             (ejecutarAccion)="emitEjecutarAccion($event)" [habilitado]="valorManual">
                         </rup>
                     </div>
                 </ng-container>
             </ng-container>
 
             <div class="d-flex align-items-center w-100">
                 <div class="w-100">
-                    <label> {{ registro.concepto.term}} </label>
                     <!-- Valor/resultado del cálculo -->
                     <ng-container *ngIf="soloValores || !valorEditable">
                         <p>
                             <ng-container *ngIf="!hasRules">
+                                <label>{{ registro.concepto.term}} </label>
                                 {{registro.valor}}
                             </ng-container>
                             <plex-badge [type]="mensaje.type" *ngIf="mensaje?.texto">
                                 {{ mensaje.texto }}
                                 <plex-button *ngIf="mensaje.add && !soloValores" (click)="addConcepto(mensaje.add)"
-                                            title="Registrar">
+                                             title="Registrar">
                                     <plex-icon name="plus"></plex-icon>
                                 </plex-button>
                             </plex-badge>
                         </p>
                     </ng-container>
                     <!-- Campo para ingreso manual del valor del registro -->
-                    <ng-container *ngIf="!soloValores && valorEditable">
-                        <plex-float *ngIf="valorManual" placeholder="Ingrese un valor" [(ngModel)]="registro.valor"
-                                    label='' name="valorCalculo" [readonly]="soloValores" (change)="emitChange2()"
-                                    [disabled]="!valorManual">
-                        </plex-float>
-                        <plex-float *ngIf="!valorManual" [(ngModel)]="registro.valor" required="true" [disabled]="true">
-                        </plex-float>
-                    </ng-container>
+                    <form #form="ngForm">
+                        <ng-container *ngIf="!soloValores && valorEditable">
+                            <plex-float [label]="params?.title || registro.concepto.term" *ngIf="valorManual"
+                                        placeholder="Ingrese un valor" [(ngModel)]="registro.valor" label=''
+                                        name="valorCalculo" [readonly]="soloValores" (change)="emitChange2()"
```

Comment:
El campo `readonly` se puede omitir, ya que mas arriba en el container se valida que sea false.
```suggestion
                                        name="valorCalculo" (change)="emitChange2()"
```

Reply:
Corregido!